flocker/node/agents/functional/test_cinder.py

```diff
@@ -113,29 +113,23 @@ def cinder_client(self):
         """
         :return: A ``cinderclient.Cinder`` instance.
         """
-        try:
-            config = get_blockdevice_config(ProviderType.openstack)
-        except InvalidConfig as e:
-            self.skipTest(str(e))
-        session = get_keystone_session(**config)
-        region = get_openstack_region_for_test()
-        return get_cinder_v1_client(session, region)
+        return self.api.cinder_volume_manager
```

Comment:
The tests below need direct access to a `python-cinderclient` `VolumeManager`.
We used to create it from scratch, but now we need to be sure that we're using the same v1 or v2 that is being used by the Flocker IBlockDeviceAPI driver.
So I return a reference to Flocker driver's `python-cinderclient` VolumeDriver instead.
Unfortunately that causes a fair amount of churn below, because the Flocker driver only has a reference to the VolumeManager, whereas we use to return a CinderClient (which *has* a VolumeManager (volumes) attribute).